src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalCompletionService.ts

```diff
@@ -224,25 +219,41 @@ export class TerminalCompletionService extends Disposable implements ITerminalCo
 			}
 			const isDirectory = kind === TerminalCompletionItemKind.Folder;
 			const pathToResource = stat.resource.fsPath.replace(cwd.fsPath, '');
-			let label = pathToResource;
+
+			let label;
+			if (!prefix.startsWith('.')) {
+				// add a dot to the beginning of the label if it doesn't already have one
+				label = '.' + pathToResource;
+			} else {
+				if (prefix.endsWith(resourceRequestConfig.pathSeparator)) {
+					// prevent a double path separator
+					label = prefix + pathToResource.substring(1);
+				} else {
+					label = prefix + pathToResource;
+				}
```

Comment:
Can we avoid this all together if we use `basename`?

Reply:
We cannot